app/check_dataset.py

```diff
@@ -179,6 +180,24 @@ def check_current(ds: DataSource, config: QCConfig) -> ResultLog:
     log.consolidate()
     return log
 
+def find_in_exceptions(row, df):
+    """
+    Find a row in a dataframe matching on state, date and exception_type
+    """
+    state, date, exception_type = itemgetter('state', 'date', 'exception_type')(row)
+    found = df.loc[(df['state'] == state) & (df['date'] == date) & (df['exception_type'] == exception_type)]
+    return (False, True)[found.size < 1]
+
+def send_anomaly_info(anomalies):
+    """
+    Warn user on found anomalies with instructions on how
```

Comment:
I think this is also where we could hook in some notifications. I'll have to dig into the slack API, I don't want to hammer the room with notifications.

Reply:
I think the place to start is a once-per-day notification based on either current or the history table that runs at around 5PM ET (after the 2nd shift).  You should be able to render the log to a text string and attach it as a snippet to a slack message so it doesn't take up a lot of screen space unless you unroll it.

You should span a background thread in either the flask App or the Pyro4 app that sends a message on startup then sleeps until 5PM ET.